docs/UserGuide.adoc

```diff
@@ -18,30 +18,34 @@ By: `Team T08-3`      Since: `Jan 2019`      Licence: `MIT`
 
 == Introduction
 
-PlanMySem is a CLI (Command Line Interface) scheduling/calendar application that targets NUS students and staff who prefer to use a desktop application for managing their schedule/calendar.
-PlanMySem automatically creates a planner that is synchronised according to the NUS academic calendar for the current semester and enables easy creation, editing and deleting of items.
+Thank you for using _PlanMySem_!
```

Comment:
Is it weird to already thank the audience for using our application? Or should we just write "Welcome to PlanMySem!"?

Reply:
Alright, will make the changes.